src/rating/rating.spec.ts

```diff
@@ -582,6 +582,33 @@ describe('ngb-rating', () => {
       expect(element.nativeElement).toHaveCssClass('ng-untouched');
     });
 
+    it('should not update template driven form by clicking disabled control', () => {
```

Comment:
I don't think this test is working at all. If I remove your fix from `rating.ts`, it is still passing